docs/index.md

```diff
@@ -16,10 +16,15 @@
 
 Please see the [guide](./witness/introduction.md) to start.
 
```

Comment:
Please also add the navigation here same as above

Reply:
okay added. formatting for guide and intro fixed.